docs/application/native/guides/compliations/watchface-complication.md

```diff
@@ -193,113 +187,111 @@ In This case, watchface supports complications only in allowed complication list
  
  
 
-# Watchface Edit with Editable feature
+## Watchface Edit with Editable Feature
 
-The watchface application can provide the changeable features(compliation, color, font..) with watchface editable APIs.
-The features can be changed by watchface editor application.
+The watchface application can provide the changeable features(such as, compliation, color, font and so on) with watchface editable APIs.

```

Comment:
Please add space after features "feature s(such as, complication...)
and there is a typo error " compliation"